.github/workflows/project-board.yml

```diff
@@ -0,0 +1,16 @@
+# This internal workflow track the status of PRs opened by the core team.
+# Open-source contributors should not worry about this workflow.
+name: 'Project Board'
+on:
+  pull_request:
+jobs:
+  add-to-project:
+    name: Add pull request to project
+    runs-on: ubuntu-latest
+    if: contains('["miguelaenlle", "mwest1066", "mylesw27", "nwalters512", "reteps"]', github.triggering_actor)
```

Comment:
`triggering_actor` uses the actor who last triggered the workflow, not the actor who initially triggered the workflow.

Reply:
What's the benefit of this? If it's non-obvious (which is the case for me), that probably belongs in a comment.

Reply:
There may be some scenarios where it's worthwhile to pull the PR onto the project board. If you would prefer we do that manually, rather than re-triggering this action, we can switch that back.